charts/observability-pipelines-worker/README.md

```diff
@@ -1,6 +1,6 @@
 # Observability Pipelines Worker
 
-![Version: 2.0.2](https://img.shields.io/badge/Version-2.0.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.0.2](https://img.shields.io/badge/AppVersion-2.0.2-informational?style=flat-square)
+![Version: 2.0.3](https://img.shields.io/badge/Version-2.0.3-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.0.2](https://img.shields.io/badge/AppVersion-2.0.2-informational?style=flat-square)
```

Comment:
Here are two more places with 2.0.2

Reply:
yeah, but the 2.0.2 are the app version, not the chart version

Reply:
This file is actually auto generated